pkg/certificate/get.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certificate
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/bestchains/bc-cli/pkg/common"
+	uhttp "github.com/bestchains/bc-cli/pkg/utils/http"
+)
+
+func NewDepositoryCertificateGetCmd(option common.Options) *cobra.Command {
+	var kid string
+
+	cmd := &cobra.Command{
+		Use:   "certificate",
+		Short: "Download the depository certificates",
+		Run: func(cmd *cobra.Command, args []string) {
+			host := viper.GetString("saas.depository.server")
```

Comment:
In depository, the `host` parameter is defined and bound to `saas.depository.server`, currently the `certificate command` does not define the `host` parameter, so the host obtained here is empty.

I think we should put the host parameter under the root cmd, And remove the `host` parameter from depository's `create` and `get` commands.

Reply:
```go
// cmd/bc-cli/main.go
func NewCmd() *cobra.Command {
   ...
    cmd.PersistentFlags().StringP("host", "http://localhost:9999", "host URL of depository server")
    _ = viper.BindPFlag("saas.depository.server", cmd.PersistentFlags().Lookup("host"))
}
```